governance.md

```diff
@@ -157,52 +161,29 @@ conflict of interest they may have. Members with a conflict of interest in a
 
-## Institutional Partners and Funding
-
-The Steering Council is the primary leadership for The Project. No outside
-institution, individual or legal entity has the ability to own, control, usurp
-or influence The Project other than by participating in The Project as
-Contributors and Council Members. However, because institutions are the primary
-funding mechanism for The Project, it is important to formally acknowledge
-institutional participation in The Project. These are Institutional Partners.
-
-An Institutional Contributor is any individual Project Contributor who
-contributes to The Project as part of their official duties at an Institutional
-Partner sustained over at least six months. Likewise, an Institutional Council
-Member is any Project Steering Council Member who contributes to The Project as
-part of their official duties at an Institutional Partner.
-
-With these definitions, an Institutional Partner is any recognized legal entity
-anywhere in the world that employs at least one Institutional Contributor or
-Institutional Council Member. Institutional Partners can be for-profit or
-non-profit entities.
-
-Institutions become eligible to become an Institutional Partner by employing
-individuals who actively contribute to The Project as part of their official
-duties. To state this another way, the only way for an Institutional Partner to
-influence The Project is by actively contributing to the open development of The
-Project, on equal terms with any other member of the community of Contributors
-and Council Members. Merely using Project Software or Services in an
-institutional context does not allow an entity to become an Institutional
-Partner. Financial gifts do not enable an entity to become an Institutional
-Partner. Once an institution becomes eligible for Institutional Partnership,
-the Steering Council must nominate and approve the Partnership.
-
-If an existing Institutional Partner no longer has a contributing employee, they
-will be given a one-year grace period for other employees to begin contributing.
-
-An Institutional Partner is free to pursue funding for their work on The Project
-through any legal means, though the Institutional Partner is obliged to inform
-the Steering Council of any such decisions. This could involve a non-profit
-organization raising money from private foundations and donors or a for-profit
-company building proprietary products and services that leverage Project
-Software and Services. Funding acquired by Institutional Partners to work on The
-Project is called Institutional Funding. However, no funding obtained by an
-Institutional Partner can override the decisions of the Steering Council. If a
-Partner has funding to do Project work and the Council decides to not pursue
-that work as a project, the Partner is free to pursue it on their own. However,
-in this situation, that part of the Partner’s work will not be under The Project
-umbrella.
```

Comment:
I have removed this entire section on "Institutional Partners". We never went through the process of identifying institutional partners, so it seems like it was not useful or relevant.